kmm/kmm-common-mvvm/src/commonMain/kotlin/com/gchristov/newsfeed/kmmcommonmvvm/CommonViewModel.kt

```diff
@@ -31,7 +32,9 @@ abstract class CommonViewModel<S : Any>(
         val currentState = _state.value
         val newState = currentState.reducer()
         if (newState != currentState) {
-            _state.postValue(newState)
+            // postValue vs setValue
+            // see: https://stackoverflow.com/questions/51299641/difference-of-setvalue-postvalue-in-mutablelivedata/51299672#51299672
+            _state.setValue(newState, false)
```

Comment:
What is the `false` here referring to? Might be nice to set argument names here so that it's clear.

Reply:
Will do in next PR